Esri/hub.js
# feat(hub-common): adding view for content edit

Issue: [https://devtopia.esri.com/dc/hub/issues/7207](https://devtopia.esri.com/dc/hub/issues/7207)

TODO:
- [x] 100% test coverage
- [x] Content is editable
- [ ] Content saves
  - [x] summary
  - [ ] featured image
  - [ ] ~~alt text~~
  - [x] categories
- [ ] Tags are being saved even when none are applied -- requires at least one

1. Description:
Creating a new view in workspaces for viewing content item's details and editing them.

1. Instructions for testing:
- visit /workspace/content/:id/details
- make edits to properties available
- save edits
- repeat with different variations

also
- test warnings

1. Closes Issues: #<7207>

1. [ ] Updated meaningful TSDoc to methods including Parameters and Returns, see [Documentation Guide](https://esri.github.io/hub-components/storybook/?path=/story/guides-documentation--page)

1. [x] used semantic commit messages
  
1. [ ] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

1. [ ] updated `peerDependencies` as needed. **CRITICAL** our automated release system can **not** be counted on to update `peerDependencies` so we _must_ do it manually in our PRs when needed. See the [updating peerDependencies](/RELEASE.md#Updating-peerDependencies) section of the release instructions for more details.
